contracts/fixed_price_oracle.vy

```diff
@@ -0,0 +1,100 @@
+struct PriceInfo:
+    price: uint256
+    lastUpdate: uint256
+
+contract ERC20():
+    def balanceOf(_owner: address) -> uint256: constant
+    def decimals() -> uint256: constant
+
+contract DaiPriceOracle:
+    def g_priceInfo() -> PriceInfo: constant
+
+PRICE_MULTIPLIER: constant(uint256) = 100000000 # 10**8
+
+# DAI oracle returns DAI/USD price multiplied by 10**18
+MIN_DAI_PRICE: constant(uint256) = 950000000000000000 # 0.95*10**18
+MAX_DAI_PRICE: constant(uint256) = 1050000000000000000 # 1.05*10**18
+# DAI_ORACLE_DIVIDER = 10**18 / PRICE_MULTIPLIER
+DAI_ORACLE_DIVIDER: constant(uint256) = 10000000000 # 10**10
+
+TokenAddressUpdated: event({token_address: indexed(address), token_index: indexed(int128)})
+
+name: public(string[16])
+owner: public(address)
+supportedTokens: public(address[5])
+allowedTokens: public(map(address, bool))
+daiAddress: public(address)
+daiOracleAddress: public(address)
+
+@public
+def __init__(dai_address: address, dai_oracle_address: address):
+    assert dai_address != ZERO_ADDRESS
+    assert dai_oracle_address != ZERO_ADDRESS
+
+    self.daiAddress = dai_address
+    self.daiOracleAddress = dai_oracle_address
+    self.allowedTokens[dai_address] = True
```

Comment:
Let's remove `allowedTokens` and just check if the address is DAI -> daiPriceOracle, if not -> 1.00

Reply:
Done